### ISSUE TITLE
Failing GitHub Django CI testing

### ISSUE THREAD

**Issue by bamaham93** [bamaham93/FS2020#17]
GitHub Django CI testing fails with the following error:
```
File "/home/runner/work/FS2020/FS2020/logic/Messaging/sms.py", line 11, in <module>
    from credentials.twilio import TWILIO_ACCOUNT_SID, TWILIO_AUTH_TOKEN
ModuleNotFoundError: No module named 'credentials.twilio'
```

**Comment by bamaham93** [comment]
Similar to #16. The fix for Issue #16 was to change the working directory /FS2020 so as to include the imported directory when run,

**Comment by bamaham93** [comment]
Just needed to mock twilio.py and faa.py for the import to not fail.
